src/components/Restoration/BusinessName.vue

```diff
@@ -0,0 +1,288 @@
+<template>
```

Comment:
This is a port of "YourCompany.vue" from Edit UI.

In Edit UI, this handles the editing of Business Name, Business Type, Name Translations, etc.

Here, it only handles Business Name (which includes NR stuff).

Reply:
When you write, "port", is this an example we could use to illustrate code duplication between the front-end UIs?  If so, it would be valuable to flag it for reference later.

Reply:
Yes, this is probably 95% duplicated code with Edit UI.

How do you propose to flag this?

Reply:
Looks like we're not discussing this with Dave until Monday, but I'd like to get this ticket wrapped up well before then.

Flag ideas -- a code comment? A tech debt ticket?

Reply:
As discussed at Standup yesterday, I will make the duplicate components shared.